Doc/library/unittest.rst

```diff
@@ -2309,17 +2309,20 @@ Class and Module Fixtures
 -------------------------
 
 Class and module level fixtures are implemented in :class:`TestSuite`. When
-the test suite encounters a test from a new class then :meth:`tearDownClass`
-from the previous class (if there is one) is called, followed by
+the test suite encounters a test that is from a different class from the previous
+test, then :meth:`tearDownClass` from the previous class is called, followed by
 :meth:`setUpClass` from the new class.
 
-Similarly if a test is from a different module from the previous test then
-``tearDownModule`` from the previous module is run, followed by
+Similarly if a test is from a different module from the previous test, then
+``tearDownModule`` from the previous module is called, followed by
 ``setUpModule`` from the new module.
 
 After all the tests have run the final ``tearDownClass`` and
 ``tearDownModule`` are run.
 
+The abovementioned calls are only executed,
+if the corresponding method or function is found.
```

Comment:
Choosing "method is found", instead of "method exists" or "class has a method", in order to hint method resolution order is in effect.